### DISCUSSION TITLE
Faint dark blobs behind achievement badges?

### DISCUSSION THREAD

**Discussion by lpulley** [community/community#33985] [Other Feature Feedback, Questions, & Ideas]
Today I noticed a faint dark "blob" behind each achievement badge:
![image](https://user-images.githubusercontent.com/7193187/192045304-b45d6bff-383a-46ee-b03d-7fdaf311da51.png)

Here is the same image run through edge detection in case you can't see it:
![image](https://user-images.githubusercontent.com/7193187/192045391-0de09fc6-d35c-47f9-a208-a779d6540dda.png)

This is not a compression artifact and actually appears on my display. I can only slightly make it out.

**Comment by zaahiral** [comment]
could have just been the way they had been exported (as the larger circle image) and be part of the way they are implemented in the site.

i.e each image has a 100px border for example(explains why the sponsor badges border looks 'smaller', and it is simply a bug that they are a different shade to the GitHub background.
